backend/examples/summa_solvency_flow.rs

```diff
@@ -57,23 +65,56 @@ async fn main() -> Result<(), Box<dyn Error>> {
     // 2. Submit Commitment
     //
     // Initialize the `Round` instance to submit the liability commitment.
-    let params_path = "ptau/hermez-raw-11";
+    let params_path = "ptau/hermez-raw-17";
     let entry_csv = "../csv/entry_16.csv";
-    let mst = MerkleSumTree::new(entry_csv).unwrap();
+    let mut entries: Vec<Entry<N_CURRENCIES>> = vec![Entry::init_empty(); N_USERS];
+    let mut cryptos = vec![Cryptocurrency::init_empty(); N_CURRENCIES];
+    parse_csv_to_entries::<&str, N_CURRENCIES>(entry_csv, &mut entries, &mut cryptos).unwrap();
+
+    let univariate_grand_sum_circuit =
+        UnivariateGrandSum::<N_USERS, N_CURRENCIES>::init(entries.to_vec());
+
+    let (params, pk, _) = generate_setup_artifacts(K, None, &univariate_grand_sum_circuit).unwrap();
+
+    // Create a proof
+    let instances = vec![Fp::one(); 1]; // This instance is necessary to verify proof on solidity verifier.
+    let (zk_snark_proof, advice_polys, _omega) = full_prover(
+        &params,
+        &pk,
+        univariate_grand_sum_circuit.clone(),
+        &[instances.clone()],
+    );
+
+    // Iterate entries and collect values to `[Vec[Fp]; N_POINTS]`
+    const FP_ARRAY: Vec<Fp> = Vec::new();
+    let mut user_data = [FP_ARRAY; N_POINTS];
```

Comment:
I think we can use the entries directly without user_data because user_data array has the same format.